assemblies/assembly_planning/main.adoc

```diff
@@ -107,10 +107,10 @@ With {PRODUCT}, you can set up a development environment to develop process-driv
 
 A development environment typically consists of one {CENTRAL} installation and at least one {KIE_SERVER} installation. {CENTRAL} enables developers to create decision processes, rules, and other artifacts. They use {KIE_SERVER} to test the artifacts that they created.
 
-A runtime environment consists of one or more {KIE_SERVER} instances with or without {CENTRAL}. {CENTRAL} enables you to manage the containers deployed on the {KIE_SERVERS}. If {CENTRAL} is not installed, you can use the headless {PRODUCT_SHORT} controller instead. Applications can interact with the {KIE_SERVER} through the REST or JMS API, or the {KIE_SERVER} Java client API.
+A runtime environment consists of one or more {KIE_SERVER} instances with or without {CENTRAL}. {CENTRAL} has an embedded {PRODUCT_SHORT} controller. If you install {CENTRAL}, use the *Execution Server* page to create and maintain containers. If you want to automate {KIE_SERVER} management without {CENTRAL}, you can use the headless {PRODUCT_SHORT} controller.
 
 .Clustered environments
-By clustering two or more computers, you have the benefits of high availability, enhanced collaboration, and load balancing. High availability decreases the chance of a loss of data when a single computer fails because data on all of the computers (nodes) of the cluster is replicated on each node of the cluster. When a computer fails, another computer fills the gap by providing a copy of the data that was on the failed computer. When the failed computer comes online again, it resumes its place in the cluster. Load balancing shares the computing load across the nodes of the cluster. Doing this improves the overall performance.
+By clustering two or more computers, you have the benefits of high availability, enhanced collaboration, and load balancing. High availability decreases the chance of a loss of data when a single computer fails. When a computer fails, another computer fills the gap by providing a copy of the data that was on the failed computer. When the failed computer comes online again, it resumes its place in the cluster. Load balancing shares the computing load across the nodes of the cluster. Doing this improves the overall performance.
 
 You can cluster both development and runtime environments. The primary benefit of clustering {PRODUCT} development environments is high availability and enhanced collaboration. The primary benefit of clustering a runtime environment is load balancing and high availability. If activity on one node of the cluster increases, that activity can be shared among the other nodes of the cluster to improve performance.
```

Comment:
Consider revising the 2nd and 3rd line to "The benefit of clustering {PRODUCT} development environments is enhanced collaboration and high availability. On the other hand, the benefit of clustering a runtime environment is load balancing and high availability.". It appeared as if secondary benefits will be listed as will next. The lines seemed disconnected.